### ISSUE TITLE
How to Define val_file_list and input_dir to run test_omnidepth.py script

### ISSUE THREAD

**Issue by Sofiane23i** [meder411/omnidepth-pytorch#7]
Hello,

I Downloaded the database mentioned in the readme, however to test the model I dont know how to define the parameters val_file_list and input_dir

**Comment by meder411** [comment]
`input_dir` is the path to the root of the dataset directory, while `val_file_list` is the path to the list of the files in validation set. When concatenated (eg `input_dir` + a row in `val_file_list`) it should provide a path to an image in the validation set.

**Comment by Sofiane23i** [comment]
thank you for your response, yes i did this but there is some problem in the code
the concatinated path is broken, it does not read the whole name (i'm debugging)
expected : '/home/listic/OmniDepth-PyTorch/Matterport3D/cLeftDown.png'
returned: '/home/listic/OmniDepth-PyTorch/Matterport3D/c'

**Comment by meder411** [comment]
Where is the code are you seeing this happen? Can you share the file and line number that produces this?

**Comment by Sofiane23i** [comment]
File "OmniDepth-PyTorch/dataset.py", line 70, in readRGBPano

**Comment by Sofiane23i** [comment]
![Screenshot from 2020-05-15 14-32-48](https://user-images.githubusercontent.com/7634734/82050870-28739100-96b9-11ea-8d0f-a39959cbe6c6.png)

**Comment by meder411** [comment]
I would check your file list to confirm that the files are not listed with spaces. You can see [here](https://github.com/meder411/OmniDepth-PyTorch/blob/master/dataset.py#L44) that all the code does is join the root and relative path that is pulled from the file. Check what the input is from [line 28](https://github.com/meder411/OmniDepth-PyTorch/blob/master/dataset.py#L28). The `loadtxt` function from numpy is what’s being used to load the file list. Your file list should be formatted as:

```
relative/path/to/rgb0.png relative/path/to/depth0.exr
relative/path/to/rgb1.png relative/path/to/depth1.exr
...
```

**Comment by Sofiane23i** [comment]
cLeftDown.png
dLeftDown.exr
nLeftDown.exr

ok thank you very much

**Comment by Sofiane23i** [comment]
even with only lowercase, i have the same problem

**Comment by Sofiane23i** [comment]
i think the problem is in 

 # Load data
        s = time.time()
        with torch.no_grad():
            for batch_num, data in enumerate(self.val_dataloader):

in omnidepth_trainer.py

**Comment by Sofiane23i** [comment]
![Screenshot from 2020-05-15 16-22-52](https://user-images.githubusercontent.com/7634734/82061041-7a6fe300-96c8-11ea-96ba-21a8eee910aa.png)

you can seeat the begining, the parsing of txt file is correct, then when we call valdataloader, the path is broken

**Comment by Sofiane23i** [comment]
> I would check your file list to confirm that the files are not listed with spaces. You can see [here](https://github.com/meder411/OmniDepth-PyTorch/blob/master/dataset.py#L44) that all the code does is join the root and relative path that is pulled from the file. Check what the input is from [line 28](https://github.com/meder411/OmniDepth-PyTorch/blob/master/dataset.py#L28). The `loadtxt` function from numpy is what’s being used to load the file list. Your file list should be formatted as:
> 
> ```
> relative/path/to/rgb0.png relative/path/to/depth0.exr
> relative/path/to/rgb1.png relative/path/to/depth1.exr
> ...
> ```

**Comment by Sofiane23i** [comment]
i tried like you said, i even copied and past your example. unfortunately the split function does not distinct between base name and extension.
```
input_dir = 'Matterport3D/' # Dataset location
val_file_list = 'newlist.txt' # List of evaluation files

and 

Matterport3D/rgb0.png Matterport3D/depth0.exr
or even absolute path
```

can you send us an example please?

**Comment by meder411** [comment]
`rgb0`, etc. are not the actual filenames. You need to provide a list of the actual files. My list of the original training and testing splits are attached here. 

In my case, the dataset is in some directory: `/path/to/omnidepth/`. This is the value of `input_dir`:

```
input_dir - '/path/to/omnidepth'
```

In that directory are the 4 subdirectories:

```
>> ls /path/to/omnidepth
Matterport  SceneNet  Stanford	SunCG
```

The relative path from `input_dir` to each input/GT pair is given in the file lists attached here. In the future, you don't have to use this setup if you have a different directory parsing strategy you'd like to use.

[original_test_split.txt](https://github.com/meder411/OmniDepth-PyTorch/files/4636273/original_test_split.txt)
[original_train_split.txt](https://github.com/meder411/OmniDepth-PyTorch/files/4636274/original_train_split.txt)
